amplitude.py

```diff
@@ -55,7 +57,7 @@ def handle (message, context):
         # This will fail if the data is not compressed.
         process_compressed(s3_object.get()["Body"].read())
```

Comment:
We need to add a `SenderThreadPool` as the first parameter right?

Reply:
Ah yeah, sorry, I didn't test the `handle` path...

Reply:
Could you make the number of threads configurable based on an environment variable? I'd like to start testing with 1 thread to ensure everything is working correctly then increase the thread count slowly

Reply:
@jbuck, env var is called `FXA_AMPLITUDE_THREAD_COUNT`.